src/Application/Actions/Person/Get.php

```diff
@@ -162,4 +123,69 @@ protected function action(Request $request, array $args): Response
             ['content-type' => 'application/json']
         );
     }
+
+    private function setTipBalances(Person $person): void
```

Comment:
Liking pulling this out of the `action` function, but I think it's still quite long and complicated. So PR to your PR to break it down further. https://github.com/thebiggive/identity/pull/183/files